ci/prow/make_config.go

```diff
@@ -531,6 +532,9 @@ func parseBasicJobConfigOverrides(data *baseProwJobTemplateData, config yaml.Map
 			}
 		case "always_run":
 			(*data).AlwaysRun = getBool(item.Value)
+		case "dot-dev":
+			(*data).PathAlias = "path_alias: knative.dev/" + (*data).RepoName
+			(*data).ExtraRefs = append((*data).ExtraRefs, " "+(*data).PathAlias)
```

Comment:
ExtraRefs don't necessarily have the same repo as the implicit ref.
Also what if there are multiple ExtraRefs (not multiple lines, but multiple ref entries)? Instead of using a list of strings as the raw config lines, why not import the actual types?

Reply:
that's a good point, however we haven't ran into that situation yet, I think we'll redo this part when needed

Reply:
Do all of your jobs have an ExtraRef? This looks like it might add a bare `path_alias: knative.dev/foo` if there is no ExtraRef specified.

Reply:
Currently ExtraRef is defined only at a single place, which makes sure all jobs get `repo`, `org` etc, and you're right that in the future this might happen, but we can always catch it during code review